### ISSUE TITLE
NTR Cryptococcal meningitis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4006]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Cryptococcal meningitis

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
https://hpo.jax.org/app/browse/term/HP:0001287 make subclass Fungal meningitis
-- Coccidiodes m. / disseminated

**Comment by pnrobinson** [comment]
good reference: https://www.thieme-connect.com/products/ejournals/html/10.1055/s-0034-1372342

**Comment by pnrobinson** [comment]
Adding term as HP_0032160
